core/src/main/kotlin/org/evomaster/core/languagemodel/LanguageModelConnector.kt

```diff
@@ -0,0 +1,241 @@
+package org.evomaster.core.languagemodel
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.google.inject.Inject
+import org.evomaster.core.EMConfig
+import org.evomaster.core.Lazy
+import org.evomaster.core.languagemodel.data.OllamaRequest
+import org.evomaster.core.languagemodel.data.OllamaResponse
+import org.evomaster.core.languagemodel.data.Prompt
+import org.evomaster.core.logging.LoggingUtil
+import java.io.DataOutputStream
+import java.net.HttpURLConnection
+import java.net.URL
+import java.util.UUID
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+import javax.annotation.PostConstruct
+import javax.annotation.PreDestroy
+import kotlin.math.min
+
+/**
+ * A utility service designed to handle large language model server
+ * related functions.
+ *
+ * Designed to work with Ollama (version 0.6.2).
+ */
+class LanguageModelConnector {
+
+    @PostConstruct
+    fun initialise() {
+        LoggingUtil.getInfoLogger().info("Initializing {}", LanguageModelConnector::class.simpleName)
+    }
+
+    @Inject
+    private lateinit var config: EMConfig
+
+    private var prompts: MutableMap<String, Prompt> = mutableMapOf()
+
+    private val objectMapper = ObjectMapper()
+
+    private var actualFixedThreadPool = 0
+
+    private lateinit var workerPool: ExecutorService
+
+    @PostConstruct
```

Comment:
why 2 `@PostConstuct` functions instead of 1?

Reply:
this was a mistake :(